### DISCUSSION TITLE
MultiView对图例的支持

### DISCUSSION THREAD

**Discussion by yuruwang** [antvis/G2Plot#2299] [Ideas]
目前多图层图表还不支持把各个图层的图例渲染在一起，希望后面能加上这块功能，例如下图，就希望区间面积图、多折线图的图例都能同时展示在图表上
![image](https://user-images.githubusercontent.com/31492974/106707137-b74f8180-662b-11eb-861d-2099fe016718.png)

**Comment by visiky** [comment]
这是 annotation 还是图例？
